exercises/luhn-trait/.meta/description.md

```diff
@@ -6,7 +6,9 @@ To get the original Luhn exercise, run
 
 > `exercism download --exercise=luhn --track=rust`
 
-To get the "Luhn: Using the From Trait" exercise, run `exercism fetch rust luhn-from`
+To get the "Luhn: Using the From Trait" exercise, run
+
+> `exercism download --exercise=luhn-from --track=rust`
```

Comment:
Instead of using blockquotes around a code span, it's better to use a code block. I.e. Instead of 

        > `exercism download --exercise=luhn-from --track=rust`


It's better to do this:

        ```shell
        exercism download --exercise=luhn-from --track=rust
        ```

This applies to all shell examples.